### PR TITLE
Add blog

### PR DESCRIPTION
**Pull request by heStudio-Network**

### CODE REVIEW

**Comment by Mister-Hope** [comment]
If your blog only contain chinese, add them in zh/demo.html only.

**Comment by heStudio-Network** [comment]
已经移除在非中文文档的链接。

File: packages/components/src/client/components/Notice.ts [hunk 1]
```diff
@@ -16,6 +16,7 @@ import type { PropType, VNode } from "vue";
 import type { NoticeLocaleOptions } from "../../shared/index.js";
 
 import "../styles/notice.scss";
+import { isLinkHttp } from "@vuepress/shared";
```
Comment (Mister-Hope) [inline packages/components/src/client/components/Notice.ts:19]:
```suggestion
```

File: packages/search-pro/src/shared/locales.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import type { RequiredLocaleConfig } from "vuepress-shared";
+import type { RequiredLocaleConfig } from "vuepress-shared/node";
```
Comment (Mister-Hope) [inline packages/search-pro/src/shared/locales.ts:1]:
```suggestion
import type { RequiredLocaleConfig } from "vuepress-shared";
```

File: docs/theme/src/zh/demo.md [hunk 3]
```diff
@@ -124,6 +124,11 @@ blog:
     url: https://whitechen233.github.io/
     source: https://github.com/whiteChen233/whitechen233.github.io
     preview: /assets/image/white.jpg
+
```
Comment (Mister-Hope) [inline docs/theme/src/zh/demo.md:127]:
```suggestion

```

**Pull request merged**